string HAPPY3="LOVE"
string HAPPY3="HAPPY3"
string LEFTATTENTION="LEFTATTENTION"
string RIGHTATTENTION="RIGHTATTENTION"
string BYELEFTHAND="BYELEFTHAND"
string BYERIGHTHAND="BYERIGHTHAND"

string motion